Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM rust:1.62.0-alpine as builder
 ARG BINARY=aggregator
-RUN apk add libc-dev
+RUN apk add libc-dev openssl-dev
```

Comment:
FYI, you'll need to set `RUSTFLAGS=-C target-feature=-crt-static` or flip on an `openssl-sys` feature for statically linking OpenSSL, otherwise you'll get issues at runtime. https://users.rust-lang.org/t/sigsegv-with-program-linked-against-openssl-in-an-alpine-container/52172/6

Reply:
Hmm, I didn't see any problems with `cargo test` or manually-running dev builds on my workstation (and release builds won't include OpenSSL at all). Maybe things have changed since that discussion?